Firestore/Example/Tests/Local/FSTLocalSerializerTests.mm

```diff
@@ -136,7 +138,7 @@ - (void)testEncodesMutationBatch {
   XCTAssertEqualObjects([self.serializer encodedMutationBatch:model], batchProto);
   FSTMutationBatch *decoded = [self.serializer decodedMutationBatch:batchProto];
   XCTAssertEqual(decoded.batchID, model.batchID);
-  XCTAssertEqualObjects(decoded.localWriteTime, model.localWriteTime);
+  XCTAssertEqual(decoded.localWriteTime, model.localWriteTime);
```

Comment:
Question: I thought `XCTAssertEqual` only works on primitives?

Reply:
It works on anything that can be compared using primitive-looking equality, which includes C++ types that define `operator==`.

Reply:
Oh, interesting. I ran into some trouble trying to do that previously and blamed it on the macro. Perhaps I was misusing it.